map_analysis_2d/ui/main_window.py

```
@@ -241,6 +246,85 @@ def _restore_last_map(self):
+            self._reset_peak_fitting_state(clear_config=True)
+            metadata = {}
+            creation_time = 'Unknown (Legacy Format)'
+            version = 'Legacy'
+
+        reversed_count = 0
+
+        if hasattr(self.map_data, 'target_wavenumbers') and self.map_data.target_wavenumbers is not None:
+            wn = self.map_data.target_wavenumbers
+            print(f"[PKL LOAD] target_wavenumbers: {wn[0]:.1f} → {wn[-1]:.1f}")
+            if len(wn) > 1 and wn[0] > wn[-1]:
+                print(f"[PKL LOAD]   Reversing target_wavenumbers to ascending order")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)